src/modules/displayLikes.js

```diff
@@ -0,0 +1,31 @@
+export const ShowLikes = async (likes) => {
+  likes.forEach((like) => {
+    const elementid = `show-${like.item_id}`;
+    const showLike = document.getElementById(elementid);
+    if (showLike !== null) {
+      const likeCount = showLike.querySelector('.like-count');
+      likeCount.innerHTML = `${like.likes}`;
+    }
+  });
+};
```

Comment:
- [ ] Can you make the pointer or mouse appear clickable when we point it to the like icon?  I am asking this because the like buttons don't show that they are clickable when you point to them. 

![Capture2](https://github.com/tatouroberto/javascriptCapstone/assets/63972670/41f9af17-dc84-437f-a287-956c2259ff3a)